WhaleSpotting/Controllers/WhaleSightingController.cs

```diff
@@ -27,4 +28,16 @@ public IActionResult GetById([FromRoute] int Id)
             return NotFound();
         }
     }
+
+    [HttpGet("")]
+    public ActionResult<List<WhaleSightingResponse>> ListApprovedSightings()
+    {
+        try{
+             return  _whaleSightingService.ListApprovedSightings();
```

Comment:
This is tiny but can we reformat this
```suggestion
        try
        {
            return  _whaleSightingService.ListApprovedSightings();
```